src/pages/documentation/adobe-audience-manager/release-notes.md

```diff
@@ -1,5 +1,19 @@
 # Release Notes
 
+## January 20, 2021
+
+### iOS Audience 2.3.0
+
+* Added tvOS compatibility
+
+## December 18, 2020
+
+### iOS Audience 2.2.0
+
+* The AEP SDKs are now distributed using XCFrameworks in order to support hardware with the new Apple M1 architecture while maintaining support for existing Intel architecture.
+  * **IMPORTANT**: Upgrading to XCFrameworks distribution requires Xcode 12.0 or newer
+  * **IMPORTANT**: If using Cocoapods, upgrading to the XCFrameworks distribution requires Cocoapods 1.10.0 or newer
+
 ## November 9, 2020
 
 ### Android Audience 1.1.0
```

Comment:
Missing the iOS Audience 2.1.0 here -  see https://github.com/AdobeDocs/aep-mobile-sdkdocs/blob/main/src/pages/documentation/adobe-audience-manager/release-notes.md

Reply:
Thanks, updated.